AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/TestSupport/HubEventsIntegrationTestBase.swift

```diff
@@ -33,8 +33,8 @@ class HubEventsIntegrationTestBase: XCTestCase {
         do {
             let configData = try Data(contentsOf: configFile)
             let amplifyConfig = try JSONDecoder().decode(AmplifyConfiguration.self, from: configData)
-            try Amplify.add(plugin: AWSAPIPlugin(modelRegistration: TestModelRegistration()))
             try Amplify.add(plugin: AWSDataStorePlugin(modelRegistration: TestModelRegistration()))
+            try Amplify.add(plugin: AWSAPIPlugin(modelRegistration: TestModelRegistration()))
```

Comment:
Not a problem, but did you reorder this because of an error, or for some other reason? The order in which we add plugins is supposed to be irrelevant, so if you encountered a problem, we should explore it immediately.

Reply:
No no, sorry for the confusion. I will reorder it back to old way